ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -122,6 +127,29 @@ public void navigateTo(String username, String title) {
         allLinks.find(Condition.exactText(title)).shouldBe(visible).click();
     }
 
+    @When("^she navigates? to the \"([^\"]*)\" page in help menu$")
+    public void navigateToHelp(String title) {
+        boolean found = false;
+        SelenideElement helpDropdownMenu = $(By.id("dropdownHelp")).shouldBe(visible);
+
+        if (!helpDropdownMenu.getAttribute("class").contains("open")) {
+            helpDropdownMenu.click();
+        }
+
+        SelenideElement dropdownElementsTable = $(By.className("dropdown-menu")).shouldBe(visible);
+        ElementsCollection dropdownElements = dropdownElementsTable.findAll(By.tagName("a"));
+
+        for (SelenideElement elem : dropdownElements) {
+            if (elem.getText().equalsIgnoreCase(title)) {
```

Comment:
The whole for-each can be substituted with filter or findBy method from ElementsCollection. Followed by shouldBe(visible).click and you'll be able to drop assert and boolean variable :)